README.md

```diff
@@ -1 +1,41 @@
-# horizen-2-doc
\ No newline at end of file
+# Website
+
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+
+## Installation
+
+```bash
+npm i
+```
+
+### Local Development
+
+```bash
+npm run start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+### Build
+
+```bash
+npm run build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
+
+Using SSH:
+
+```bash
+ USE_SSH=true yarn deploy
+```
+
+Not using SSH:
+
+```bash
+ GIT_USER=<Your GitHub username> yarn deploy
+```
+
+If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
```

Comment:
Do we use these commands? If not, it can be removed. Otherwise let's keep consistent and use npm here too.